volatility3/framework/contexts/__init__.py

```diff
@@ -462,3 +462,149 @@ def __init__(
         Module.__init__(
             self, context, name, layer_name, offset, symbol_table_name, layer_name
         )
+
+
+class MacOSKernelCacheSupportModule(Module):
```

Comment:
It's actually not the Module/ModuleInterface that needs to define this, but the ModuleRequirement, so this might require changing mac plugins to require a mac module?  5:S